File: Programs/5 Languages/HW01.cpp
#include <iostream>
#include <math.h>

using namespace std;

int main(){
	int num;
	cout << "Hello, World!" << endl;
	cout << "Input a number: ";
	cin >> num;
	cout << "Your entered number: " << num << endl;
	
	int i, cnt = 0;
	
	for(i = 1; i < num; i++){
		if( num%i == 0 )
			cnt++; 
	}
	
	if(cnt > 1){
		cout << num << " is not a prime number" << endl;
	} else {
		cout << num << " is a prime number"	<< endl;		
	}
	
	float x;
	x = pow(num, 2) + ((log10(num) - sin(num))/ sqrt(num));
	cout << "f(" << num << ") is " << x << endl;
	
	return 0;
}

File: Programs/5 Languages/HW01.py
import math

print ('Hello, World!')

num = int(input("Enter a number: "))
print ("You entered: ", num)

if num > 1:
   for i in range(2,num):
       if (num % i) == 0:
           print(num," is not a prime number")
           print(i,"times",num//i,"is",num)
           break
   else:
       print (num," is a prime number")
else:
   print (num," is not a prime number")

equate = num ** 2 + ((math.log10(num) - math.sin(num))/ math.sqrt(num))
print ("The equated expression is: ", equate)